gmso/formats/top.py

```diff
@@ -174,29 +151,7 @@ def write_top(top, filename):
         )
```

Comment:
Small change but the dihedral comment on line 150 should match the dihedral potential form.

Reply:
Good point. How should we handle this in the case that we have multiple dihedral types?

Reply:
Do you mean in the case where both harmonic and RB dihedrals would be present in the system?

Reply:
Yeah. This would be a lot easier if there was an easier way to group dihedrals by type. Another improvement would be to have a function that returns the potential type name if you pass it a potential (kind of like what `_write_connection` does).